dropwizard/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaSettings.java

```diff
@@ -0,0 +1,972 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright 2012-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linecorp.armeria.dropwizard;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import javax.annotation.Nullable;
+
+import com.google.common.collect.ImmutableList;
+
+import com.linecorp.armeria.common.SessionProtocol;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.VirtualHostBuilder;
+
+import io.netty.handler.ssl.ClientAuth;
+import io.netty.handler.ssl.SslProvider;
+
+/**
+ * Settings for armeria servers, e.g.,
+ * <pre>{@code
+ * metricsPath: "/internal/metrics"
```

Comment:
I wonder if we need this feature or not because this is probably not part of server settings. Thoughts?

Reply:
Metrics should be served under the adminContext / servlet by default

Reply:
> I wonder if we need this feature or not because this is probably not part of server settings. Thoughts?

I agree. It is an additional feature, not a server setting.

> Metrics should be served under the adminContext / servlet by default

That's a good point. I checked out the '/admin/metrics' on my local Armeria Dropwizard server. Let me drop `metricsPath`.